src/main/java/com/google/sps/servlets/MetadataStore.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.util.List;
+
+/**
+ * An interface that defines methods for interacting
+ * with some type of external database (ex: Datastore, Firestore, SQL)
+ */
+public interface MetadataStore {
+  public void storeData(Metadata data);
```

Comment:
nit: I'd be nice to add a comment on each method on what it's supposed to do -- although they may be guessable from their names.

Reply:
I've added brief comments to each one.